.github/actions/functionnal-test-stdout/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Functional Tests
+description: Run Functional Tests
+runs:
+  using: "composite"
+  steps:
+    - run: npm install --silent --unsafe-perm
+      shell: bash
```

Comment:
Is there a way to not specify `bash` all the time?

Reply:
Not that i now of, i'll keep looking and update if i found something

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#defaultsrun

Reply:
Thats about the workflow, not a composite action.yml file

The docs about it is https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-composite-run-steps-action and i'll have a deeper look into it, but in their example all of them have duplicate `shell:bash`